### DISCUSSION TITLE
Welcome to Python-projects Discussions!

### DISCUSSION THREAD

**Discussion by AaravAtGit** [AaravAtGit/Python-projects#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tejaschigateri15** [comment]
which is the best way to start learning artificial intelligence using python
